samples/cpp/benchmark_app/inputs_filling.cpp

```diff
@@ -261,6 +261,27 @@ ov::Tensor create_tensor_random(const benchmark_app::InputInfo& inputInfo,
     return tensor;
 }
 
+ov::Tensor create_tensor_random_4bit(const benchmark_app::InputInfo& inputInfo,
```

Comment:
Mapping to <uint8_t>

Reply:
You didn't update Python analogs for `create_tensor_from_numpy()` and `create_tensor_from_binary()`. Do they work?

BTW, here's how you can generate test data: https://github.com/openvinotoolkit/openvino/pull/14021#issuecomment-1405916329

Reply:
create_tensor_from_binary test pass. 
But create_tensor_from_numpy still test fail. 
Here is my script about generating test data:
```
def save_npy(shape, out_fn):
    z = np.zeros(shape, dtype=np.uint8)
    np.save(out_fn, z)
save_bin((64), '128_u4.npy')
```
@Wovchena , Actually, we don't need to implment create_tensor_from_numpy, because numpy don't support i4/u4, minimal data type is np.int8. So I think create_tensor_from_binary is enough, what's your idea?

Reply:
Yeah, makes sense. Can you raise an exception if numpy reading happens to be requested with i4 model inputs.

Reply:
updated.